src/cli.rs

```diff
@@ -270,6 +271,14 @@ pub struct Opt {
     #[structopt(long = "show-syntax-themes")]
     pub show_syntax_themes: bool,
 
+    /// Show all available delta themes, each with an example of highlighted diff output.
+    /// If diff output is supplied on standard input then this will be used for the demo. For
+    /// example: `git show --color=always | delta --show-themes`.
```

Comment:
I don't think `--color=always` is needed here.

Reply:
👍 removed.